runtime/flangrti/iostdinit.c

```diff
@@ -153,7 +153,7 @@ __io_ferror(void *p)
 int
 __io_getfd(void *fp)
 {
-  return (((FILE *)fp)->_fileno);
+  return fileno((FILE *)fp);
```

Comment:
Good luck with that! I spotted this years before, and proposed change was rejected.

Reply:
Any particular reason it was rejected? Does it apply now?

Reply:
AFAIR one of the building environments maintained by the original project owners did not provide `fileno()` function. A change like that was breaking compatibility with that platform.

Reply:
@s-u Would it be possible to conditionally make this change for Mac?

Reply:
Obviously, I could, but why leave bad code in there? Accessing system's internal structure that are supposed to be opaque will break on any decent system, not just macOS (hint, hint, that's why the name starts with an underscore!).

Reply:
OK. let us use your version. If we see a platform where there is an issue then we will change.